.github/workflows/build_and_test.yml

```diff
@@ -65,19 +65,21 @@ jobs:
       run: |
         ./.github/workflows/build-test nomypy
     - name: Set up Python 3.9
-      if: github.event_name == 'pull_request'
+      if: github.event_name == 'pull_request' || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel')
```

Comment:
do we still need the wheel/* branch check here?

Reply:
Yeah, so that we can build wheels from a `wheel/**` branch.